### ISSUE TITLE
Checkbox Cell Edit fails on Safari

### ISSUE THREAD

**Issue by BrianSHenderson** [AllenFang/react-bootstrap-table#1482]
It seems impossible to select the checkbox with version 3.4.0, has anyone else found this?

**Comment by AllenFang** [comment]
@BrianSHenderson whats version of your Safari?

**Comment by BrianSHenderson** [comment]
v10.1.1

**Comment by AllenFang** [comment]
Thanks, could you please help me to check [this page](https://allenfang.github.io/react-bootstrap-table/example.html#selection) if it work well on your Safari?

**Comment by BrianSHenderson** [comment]
The examples on that page work well but are doing something slightly different.  I have a table where some cells can be edited whereas other columns are checkboxes and I use the following settings.
    var cellEditProp = {
      mode: "click",
      blurToSave: true,
       afterSaveCell: onAfterSaveCell
    };

Would I need to reorganise the page into allowing to select multiple rows and then running the code?

**Comment by AllenFang** [comment]
@BrianSHenderson, yes, you can try it. and could you please show me your `selectRow` props?

BTW, in your table, you enable selection and cell edit, so you means when you click the checkbox inside the selection column is not work well?

**Comment by BrianSHenderson** [comment]
I wasn't using the selectRow functionality at all, I had multiple columns responding to the cellEditProp (text fields and checkboxes), perhaps using SelectRow would have been better.

**Comment by AllenFang** [comment]
@BrianSHenderson thanks, I'll check this, it seems like a bug.
